routes/user_page.py

```diff
@@ -0,0 +1,79 @@
+from flask import Blueprint, g, request, abort, jsonify
+from flask_jwt_extended import (
+    get_jwt_identity, jwt_required
+)
+from models.base import get_session
+from models.User import User
+from models.Place import Place
+from models.UserFavoritePlace import UserFavoritePlace
+from flask_expects_json import expects_json
+
+
+user_page_blueprint = Blueprint('user_page', __name__)
+
+
+put_favorite_schema = {
+    'type': 'object',
+    'properties': {
+        'place_id': {'type': 'integer'},
+    },
+    'required': ['place_id']
+}
+
+
+@user_page_blueprint.route('/user_page/favorite_place/', methods=['POST'])
```

Comment:
Lets change URL to something like /user/favorite. I think it's not good to name API routes with 'page' word.